### PR TITLE
Add CentOS Stream 8

### PR DESCRIPTION
**Pull request by krionbsd**

### CODE REVIEW

File: os-images/AWS/centos/centosstream-8-x86-64-us-west-2.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "_comment": "Build with `packer build -var-file=centos-8-x86-64-us-west-2.json centos.json`",
```
Comment (bryceml) [inline os-images/AWS/centos/centosstream-8-x86-64-us-west-2.json:2]: should be centosstream

File: os-images/Docker/centos/centosstream-8-vars.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "_comment": "Build with `packer build -var-file=centos-8-vars.json centos.json`",
```
Comment (bryceml) [inline os-images/Docker/centos/centosstream-8-vars.json:2]: should be centosstream

File: os-images/AWS/centos/centosstream-8-x86-64-us-west-2.json (hunk 1 above)
Comment (krionbsd) [reply os-images/AWS/centos/centosstream-8-x86-64-us-west-2.json]: Fixed

File: os-images/Docker/centos/centosstream-8-vars.json (hunk 2 above)
Comment (krionbsd) [reply os-images/Docker/centos/centosstream-8-vars.json]: Fixed

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**